### PR TITLE
Feature/google auth button

### LINKED ISSUE #30: Finish a google auth button like the Facebook one
Actually we can connect using FB. 

Can we do the same with Google ? check the package `react-google-login` to generate the google link.

### PR DESCRIPTION
**Pull request by analuspi**
Fix #30 

But I don't know how to integrate <GoogleLogin /> to <AiFillGoogleCircle />

### CODE REVIEW

**Comment by julbrs** [comment]
@analuspi I will look a that today ! thansk !

File: src/components/Navigation/Login.js [hunk 1]
```diff
@@ -50,6 +55,12 @@ const Login = (props) => {
 
           <Navbar.Item>
             <AiFillGoogleCircle size="2em" title="Pas encore disponible..."/>
+            <GoogleLogin
+                            clientId="450944417353-pe3vqggjq8ouakahnr8futu7eju6jvm7.apps.googleusercontent.com"
```
Comment (julbrs) [inline src/components/Navigation/Login.js]: Where do you find this clientId ? It is not the one defined here ? https://github.com/orgs/montessori-ressources/teams/core

**Comment by julbrs** [comment]
I have added a commit (6c2d17594e1cef33af4818820c57c5f35d39b3aa) and also some change on the API because google wasn't working 😄  now it is ok ! I let you merge ?

**Review by analuspi — COMMENTED** [review]
I created a client ID in my Google account (I didn't know we had one already). I think it wasn't working because I've just authorized local host (since it worked well in my computer)

**Pull request merged**